source/reference/minio-mc-admin/mc-admin-info.rst

```diff
@@ -21,21 +21,25 @@ for each MinIO server in the deployment.
 
 .. end-mc-admin-info-desc
 
+.. versionchanged:: RELEASE.2023-07-18T17-49-40Z
+
+   MinIO now reports the number of delete markers on the deployment.
```

Comment:
```suggestion
   The output includes the number of delete markers on the deployment.
```

Kind of awkward to have this here - we don't make comments for when anything else was added.

Maybe add it after the example instead?

Reply:
Just deleting it. The updated output includes delete marker count now. That's good enough.